conversions/decimal_to_any.py

```diff
@@ -0,0 +1,82 @@
+"""Convert a positive Decimal Number to Any Other Representation"""
+
+
+def decimal_to_any(num: int, base: int) -> str:
+
+    """
+        Convert a Integer Decimal Number to a Binary Number as str.
+        >>> decimal_to_any(0, 2)
+        '0'
+        >>> decimal_to_any(5, 4)
+        '11'
+        >>> decimal_to_any(20, 3)
+        '202'
+        >>> decimal_to_any(58, 16)
+        '3A'
+        >>> # negatives will error
+        >>> decimal_to_any(-45, 8) # doctest: +ELLIPSIS
+        Traceback (most recent call last):
+        ...
+        ValueError: parameter must be positive int
+        >>> # floats will error
+        >>> decimal_to_any(34.4, 6) # doctest: +ELLIPSIS
+        Traceback (most recent call last):
+        ...
+        TypeError: 'float' object cannot be interpreted as an integer
```

Comment:
The exception text should match the exception raised by `int(34.4, base=6)`
https://docs.python.org/3.4/library/functions.html?highlight=open#int